### PR TITLE
Reflows charts in next animation frame after render

### LINKED ISSUE #100: Reflow highchart after initial display
When a chart is displayed inside of a container that has not yet determined its dimensions, it may be improperly sized. This can be fixed by immediately reflowing the chart in the next animation frame. Please add the following code after [chartsFactory.jsx:36](https://github.com/kirjs/react-highcharts/blob/master/src/chartsFactory.jsx#L36):

```javascript
let highchart = this.chart
requestAnimationFrame(function(){
	highchart.reflow()
})
```

### PR DESCRIPTION
**Pull request by kenaniah**
Fixes #100

### CODE REVIEW

File: src/chartsFactory.jsx [hunk 1]
```diff
@@ -34,6 +34,10 @@ if(typeof Highcharts === 'undefined'){
             renderTo: this.refs.chart
           }
         });
+        let highchart = this.chart
+        requestAnimationFrame(function(){
+          highchart.reflow()
+        })
```
Comment (kirjs) [inline src/chartsFactory.jsx]: Can you use an arrow function here to avoid creating an extra variable?

**Comment by kirjs** [comment]
Hi Kenaniah, thanks a lot for the PR, looks like it broke the tests. 
Feel free to take a look, or I can do that once I have some spare time.

**Comment by kenaniah** [comment]
Looks like JSDOM doesn't provide `window.requestAnimationFrame()`. There's a polyfill (https://gist.github.com/paulirish/1579671) that can be used for testing purposes, but not sure if/how you wanted to include that in the project.

**Comment by kirjs** [comment]
Hi Kenaniah, I finally have some time to look at the PR. 
Just wanted to clarify: would we want to make thing behavior configurable for any reason?
e.g. does it have any side effects?

**Comment by kenaniah** [comment]
No side effects to my knowledge. Animations still appear to function normally.

**Comment by kirjs** [comment]
I merged in the change manually, it should be available starting with  v8.0.2, thanks a lot!

**Comment by marian-c** [comment]
When using `highcharts-solid-gauge`, the initial render animation will stop happening. 

I think the `reflow()` call should be the responsibility of the user, or the user should be able turn the auto `reflow()` call off, from a prop.

notifying @kenaniah, @kirjs

**Comment by kirjs** [comment]
https://github.com/kirjs/publish-highcharts-modules/issues/7 seems related?

**Comment by marian-c** [comment]
Hey, yup it's referring to the same problem.

**Comment by kirjs** [comment]
OK, reflow is optional now.

**Pull request closed without merging**